win32ss/user/winsrv/usersrv/harderror.c

```diff
@@ -583,12 +628,24 @@ UserServerHardError(
         return;
     }
 
-    /* Display the message box */
-    Message->Response = UserpMessageBox(TextU.Buffer,
-                                        CaptionU.Buffer,
-                                        Message->ValidResponseOptions,
-                                        (ULONG)Message->Status >> 30);
-
+    if ((Message->Status & (OptionOkNoWait << 30)) != 0)
+    {
+        DPRINT1("Got balloon hard error!\n");
+        /* Display the balloon */
+        UserpShowInformationBalloon(TextU.Buffer,
+                                    CaptionU.Buffer,
+                                    Parameters[2],
```

Comment:
Shouldn't it be "Message->ValidResponseOptions" instead?

Reply:
No, its the parameter name that is incorrect.